App.js

```diff
@@ -15,7 +15,17 @@ export default class App extends Component {
     }
   }
 
-  componentDidMount() {
+  getStartLocation = () => {
+    navigator.geolocation.getCurrentPosition(
+      position => {
+        this.setState({ currentLatitude: position.coords.latitude, currentLongitude: position.coords.longitude });
+      },
+      error => console.log(error),
+      { enableHighAccuracy: true, timeout: 20000, maximumAge: 1000 }
+    );
+  }
```

Comment:
The watchPosition tracking wasn't as responsive as getCurrentPosition so I added it back in to get the user's initial position so that the user doesn't have to wait as long